jax/_src/compiler.py

```diff
@@ -358,11 +384,6 @@ def _compile_and_write_autotune_config(
   )
 
   if os.path.exists(autotune_tmp_file):
-    logger.debug(
```

Comment:
It still looks like this PR is removing a bunch of logging, which seems like an oversight?

Reply:
They should all be resolved! For some reason when I merged upstream main into my branch it just omitted all the logging statements, I'm not sure why.